data/tutorials/gs_00_up_and_running.md

```diff
@@ -11,88 +11,165 @@ date: 2021-05-27T21:07:30-00:00
+
+```shell
+$ opam switch create 5.0.0
+$ eval $(opam env)
 ```
+
+Check that the installation was succuessful by running `which ocaml` and `ocaml -version`. The line beneath the $ command shows the desired output for both the OCaml version and the toplevel version (installed specifically with the above `switch` command):
+
+```shell
 $ which ocaml
-/Users/frank/.opam/4.14.0/bin/ocaml
+/Users/frank/.opam/5.0.0/bin/ocaml
 
 $ ocaml -version
-The OCaml toplevel, version 4.14.0
+The OCaml toplevel, version 5.0.0
 ```
 
-As an alternative **for either Linux or macOS**, a binary distribution of Opam is
-available:
+We will learn about the OCaml toplevel and other installed tools in the next section.
 
-```
-sh <(curl -sL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh)
-```
+In case you are not satisfied with the OCaml version of your system switch, you can change the version with `opam switch create <version_here>`. More information can be found on the [official website](https://opam.ocaml.org/).
+
+
+### The OCaml Base Tools
+
+OCaml is installed in an opam switch, which, among others, bring the following
+programs:
+
+- A "toplevel," which can be called with the `ocaml` command. It consists of a read-eval-print loop ([REPL](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop)), similar to the `python` or `node` command, and can be handy to quickly try the language. The user interface of OCaml is very basic, but when we install the UTop package in the following section, we'll have an improved and easier-to-use REPL.
+
+- A compiler to **native code**, called `ocamlopt`. It creates executables that can be executed directly on your system.
+
+- A compiler to **bytecode**, called `ocamlc`. It creates executables that can be interpreted by a variety of runtime environments, making it more flexible.
+
+Although this is theoretically all we need to write OCaml code, it is not a complete and comfortable development environment.
+
+## Setting Up Development Tools
 
-### For Windows
+We will now install everything we need to get a complete development environment, which includes:
 
-OCaml on Windows is still a work in progress. In the meantime, if you only
-need to _run_ OCaml programs on a Windows machine, then the simplest solution is to use the Windows Subsystem for Linux 2 (WSL2). WSL2 is a feature that allows Linux programs to run directly on Windows. WSL2 is substantially easier and faster to use than WSL1. Microsoft has comprehensive installation steps for [setting up WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+- Dune, a fast and full-featured build system for OCaml
+- Merlin (the backend) and `ocaml-lsp-server` (OCaml's Language Server Protocol) to provide editors with many useful features such as "jump to definition"
```

Comment:
nitpick: a ocaml-lsp-server is a "language server" (or maybe a "LSP server") rather than a "language server protocol".